decidim-proposals/spec/system/vote_proposal_spec.rb

```diff
@@ -372,6 +421,7 @@ def expect_page_not_to_include_votes
           within proposal_element do
             within ".card__support", match: :first do
               click_button "Vote"
+              sleep(5)
```

Comment:
Meh

Reply:
```suggestion
            end
            
            within ".card__support", match: :first do
              expect(page).to have_content("1 VOTE")
            end
```

Reply:
Maybe creating a new `within` and moving the expectation there avoids the sleep.

Reply:
Oh, that sleep is totally there by mistake!